services/ui-src/src/routes/ReviewSubmit/ReviewSubmit.tsx

```diff
@@ -1,29 +1,215 @@
+  const reportYear = "2022";
+  const reportKey = `${state}${reportYear}`;
+
+  // TODO: get real program name per report
+  const programName = "tempName";
+
+  const reportDetails = {
+    key: reportKey,
+    programName: programName,
+  };
+
+  useEffect(() => {
+    fetchReportStatus(reportDetails);
+  }, []);
+
+  const submitForm = () => {
+    // TODO: Add check to make sure user filled out the form
+    if (userRole === UserRoles.STATE_USER || userRole === UserRoles.STATE_REP) {
+      updateReportStatus(reportDetails, ReportStatus.COMPLETED);
+    }
+    // TODO: Handle Error for other users coming on to the page
+    onClose();
+  };
+
   return (
     <ReportPage data-testid="review-and-submit-view">
+      <Flex sx={sx.pageContainer}>
+        <Sidebar />
+        {reportStatus?.status == ReportStatus.COMPLETED ? (
+          <SuccessMessage
+            programName={programName}
+            date={reportStatus?.lastAltered}
+            givenName={user?.given_name}
+            familyName={user?.family_name}
+          />
+        ) : (
+          <ReadyToSubmit
+            submitForm={submitForm}
+            isOpen={isOpen}
+            onOpen={onOpen}
+            onClose={onClose}
+          />
+        )}
+      </Flex>
+    </ReportPage>
+  );
+};
+
+const ReadyToSubmit = ({
+  submitForm,
+  isOpen,
+  onOpen,
+  onClose,
+}: ReadyToSubmitProps) => {
+  const { intro, modal, pageLink } = reviewVerbiage;
+
+  return (
+    <Flex sx={sx.contentContainer} data-testid="ready-view">
       <Box sx={sx.leadTextBox}>
         <Heading as="h1" sx={sx.headerText}>
           {intro.header}
         </Heading>
-        <Text>{intro.body}</Text>
+        <Box sx={sx.infoTextBox}>
+          <Text sx={sx.infoHeading}>{intro.infoHeader}</Text>
+          <Text>{intro.info}</Text>
+        </Box>
       </Box>
-    </ReportPage>
+      <Flex sx={sx.submitContainer}>
+        <Button
+          type="submit"
+          colorScheme="colorSchemes.primary"
+          rightIcon={<Icon icon="arrowRight" />}
+          onClick={() => onOpen()}
+          data-testid="submit-button"
+        >
+          {pageLink.text}
+        </Button>
+      </Flex>
+      <Modal
+        actionFunction={() => submitForm()}
+        modalState={{
+          isOpen,
+          onClose,
+        }}
+        content={modal}
+      />
+    </Flex>
   );
 };
 
+interface ReadyToSubmitProps {
+  submitForm: Function;
+  isOpen: boolean;
+  onOpen: Function;
+  onClose: Function;
+}
+
+export const SuccessMessage = ({
+  programName,
+  date,
+  givenName,
+  familyName,
+}: SuccessMessageProps) => {
+  const { intro } = successVerbiage;
+  const utcDateToReadableDate = new Intl.DateTimeFormat("en-US", {
+    dateStyle: "full",
+  }).format(date);
+  const submittedDate = `was submitted on ${utcDateToReadableDate}`;
+  const submittersName = ` by ${givenName} ${familyName}`;
+  return (
+    <Flex sx={sx.contentContainer} data-testid="submitted-view">
```

Comment:
For the test ids like this, we can probably just check for actual text in the dom, like "Successfully submitted" or whatever it says.